go/pkg/services/logcollector/server.go

```diff
@@ -475,6 +480,65 @@ func (s *Server) StopLog(ctx context.Context, request *protologcollector.StopLog
 	return s.successfulBaseResponse(), nil
 }
 
+// DeleteLogs deletes the log file for a given run id or project
+func (s *Server) DeleteLogs(ctx context.Context, request *protologcollector.StopLogsRequest) (*protologcollector.BaseResponse, error) {
+	if !s.isChief {
+		s.Logger.DebugWithCtx(ctx,
+			"Server is not the chief, ignoring delete logs request",
+			"project", request.Project,
+			"numRunIDs", len(request.RunUIDs))
+		return s.successfulBaseResponse(), nil
+	}
+
+	// validate project name
+	if request.Project == "" {
+		message := "Project name must be provided"
+		s.Logger.ErrorWithCtx(ctx, message)
+		return &protologcollector.BaseResponse{
+			Success:      false,
+			ErrorCode:    0,
+			ErrorMessage: message,
+		}, errors.New(message)
+	}
+
+	s.Logger.DebugWithCtx(ctx,
+		"Deleting logs",
+		"project", request.Project,
+		"numRunIDs", len(request.RunUIDs))
+
+	// if no run uids were provided, delete the entire project's logs
+	if len(request.RunUIDs) == 0 {
```

Comment:
we usually use `"*"` to mark we want 'all' to avoid deleting everything bu accident

Reply:
It is a list though.
Do you mean sending something like `["*"]`?